packages/jest/src/__tests__/matchers.test.tsx

```diff
@@ -1,57 +1,88 @@
 import { render } from '@testing-library/react';
 import React from 'react';
-import '@compiled/css-in-js';
-
-it('should detect styles', () => {
-  const { getByText } = render(
-    <div
-      css={{
-        fontSize: '12px',
-      }}>
-      hello world
-    </div>
-  );
-
-  expect(getByText('hello world')).toHaveCompiledCss('font-size', '12px');
-});
+import { styled } from '@compiled/css-in-js';
 
-it('should detect missing styles', () => {
-  const { getByText } = render(<div css={{ fontSize: '12px' }}>hello world</div>);
+describe('toHaveCompliedCss', () => {
+  afterEach(() => {
+    document.getElementsByTagName('html')[0].innerHTML = '';
```

Comment:
Noticed that we need to clear the head after every test because styles are left behind after every test :(

Reply:
is that really a problem?

Reply:
maybe not, if we only attempt to get styles for the provided css classes 🤔

Reply:
Updated code to narrow down to properties contained within the supplied classes